packages/cli/src/commands/doctor/types.ts

```diff
@@ -89,13 +89,19 @@ export type HealthCheckInterface = {
   description?: string;
   getDiagnostics: (
     environmentInfo: EnvironmentInfo,
-  ) => Promise<{version?: string; needsToBeFixed: boolean | string}>;
+  ) => Promise<{
+    version?: string;
+    versionRange?: string;
```

Comment:
shouldn't we require versions and ranges from all health checks?

Reply:
Some of them are not based on comparing versions, just checking if they are there or not (e.g. `ios-deploy` and `cocoapods`).